src/content/contribute/writing-a-loader.md

```diff
@@ -208,14 +209,22 @@ For instance, the `sass-loader` [specifies `node-sass`](https://github.com/webpa
 So you've written a loader, followed the guidelines above, and have it set up to run locally. What's next? Let's go through a simple unit testing example to ensure our loader is working the way we expect. We'll be using the [Jest](https://facebook.github.io/jest/) framework to do this. We'll also install `babel-jest` and some presets that will allow us to use the `import` / `export` and `async` / `await`. Let's start by installing and saving these as a `devDependencies`:
 
 ``` bash
-npm i --save-dev jest babel-jest babel-preset-es2015 babel-preset-stage-0
+npm i --save-dev jest babel-jest babel-preset-env
 ```
 
 __.babelrc__
 
 ``` json
 {
-  "presets": [ "es2015", "stage-0" ]
+  "presets": [
+    "env",
+    {
+      "targets": {
+        "node": "4"
+      },
+      "modules": false
+    }
+  ]
```

Comment:
I had to refactor this a bit to get it working in my local demo:

``` json
{
  "presets": [[
    "env",
    {
      "targets": {
        "node": 4
      }
    }
  ]]
}
```

The two changes are...

- Wrapping the env section and options in a nested array, I think this is the only way babel knows how to associate the options with the preset.
- Removed `"modules": false` as it makes the `import` statements in `loader.test.js` and any other `import` statements used in the `/test` modules fail.

Once I made those changes, everything ran fine again.